src/main/resources/resources/DefaultJahiaTemplates_en.properties

```diff
@@ -308,7 +308,7 @@ jnt_user.yourPreferences=Your preferences
 jnt_video=Video
 jnt_video.autoplay=Auto play
 jnt_video.autostart=Auto start
-jnt_video.forceFlashPlayer=Force Flash Player (Deprecated)
+jnt_video.forceFlashPlayer=Force Flash Player
```

Comment:
is it related to here ?

Reply:
Nope, from a different task, bad merge after rebase, should be fixed now